synapse/module_api/__init__.py

```diff
@@ -346,25 +335,8 @@ async def create_and_send_event_into_room(
         Raises:
             SynapseError if the event was not allowed.
         """
-        if event_type == EventTypes.Member:
-            raise Exception(
-                "Synapse modules are not currently able to send m.room.member events"
-            )
```

Comment:
we probably need to keep this?

Reply:
Ah, should've mentioned. It's already enforced in `create_and_send_nonmember_event`, though it raises a `SynapseError` versus a generic `Exception`.

Reply:
so it is. my bad.